libs/sdk-ui-kit/src/SettingWidget/tests/StylingSettingWidget.test.tsx

```diff
@@ -62,14 +62,12 @@ describe("StylingPicker", () => {
         expect(screen.getByText("Second theme")).toBeInTheDocument();
     });
 
-    it("should render empty message and no footer buttons when no custom items are provided", () => {
+    it("should render empty message when no custom items are provided", () => {
         renderComponent({
             customItems: [],
         });
 
         expect(screen.getByLabelText("empty-message-test")).toBeInTheDocument();
-        expect(screen.queryByText("Cancel")).not.toBeInTheDocument();
```

Comment:
Should we just change the expectations here, rather than deleting them? Something like below:

```
expect(screen.queryByText("Cancel")).not.toBeInTheDocument();
expect(screen.queryByText("Apply")).not.toBeInTheDocument();
```

↓

```
expect(screen.queryByText("Cancel")).toBeInTheDocument();
expect(screen.queryByText("Apply")).toBeInTheDocument();
```